apis/go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/go-openapi/spec v0.19.5
 	github.com/onsi/ginkgo v1.14.2
 	github.com/onsi/gomega v1.10.4
+	github.com/robfig/cron/v3 v3.0.1
```

Comment:
can we remove the dependency towards the cron in the apispec?
I think we only really need it in the landscaper/deployer code.

Reply:
It's currently required by the validation of the ContinuousReconcileSpec: https://github.com/Diaphteiros/landscaper/blob/continuous-reconcile/apis/deployer/utils/continuousreconcile/validation/validation.go#L35 ...

Reply:
ok then let's keep. I think the validation more useful than having one less dependency.